### ISSUE TITLE
[Jaet] Add joy

### ISSUE THREAD

**Issue by waghanza** [the-benchmarker/web-frameworks#2870]
Hi @swlkr ,

Would you mind to add (or consent that we add) [joy](https://github.com/joy-framework/joy) in `here` ?

This project compare performance based on 3 routes :

+ `GET` `/`, **SHOULD** return a successful status code and an empty string
+ `GET` `/user/:id`, **SHOULD** return a successful status code and the id
+ `POST` `/user`, **SHOULD** return a successful status code and an empty string

Regards,

**Comment by swlkr** [comment]
We could, but it's not going to be very fast. The server is currently single threaded 😬 

There are some changes coming to janet which could boost performance quite a bit with an event loop/threads, but until then it will be slow 😢

**Comment by waghanza** [comment]
I don't think it could only be `here` if fast, their is also the code expressiveness, but however, if you think it is better to delay that, you could close and re-open later
